test/integration/node-specific/client_close.test.ts

```diff
@@ -584,7 +699,57 @@ describe.skip('MongoClient.close() Integration', () => {
 
   describe('Server resource: Cursor', () => {
     describe('after cursors are created', () => {
-      it.skip('all active server-side cursors are closed by client.close()', async function () {});
+      let client;
+      let coll;
+      let cursor;
+      let utilClient;
+
+      beforeEach(async function () {
+        client = this.configuration.newClient();
+        utilClient = this.configuration.newClient();
+        await client.connect();
+        coll = await client.db('db').collection('coll', { capped: true, size: 1_000_000 });
+      });
+
+      afterEach(async function () {
+        await utilClient?.close();
+        await client?.close();
+        await cursor?.close();
+      });
+
+      it.skip('all active server-side cursors are closed by client.close()', async function () {
+        const getCursors = async () => {
+          const res = await utilClient
+            .db()
+            .admin()
+            .command({
+              aggregate: 1,
+              cursor: {},
+              pipeline: [{ $currentOp: { idleCursors: true } }]
+            });
+          return res.cursor.firstBatch.filter(
+            r => r.type === 'idleCursor' || (r.type === 'op' && r.desc === 'getMore')
+          );
+        };
+
+        await coll.insertMany([{ a: 1 }, { b: 2 }, { c: 3 }]);
```

Comment:
I think the insert should probably be in the before since it isn't part of the test

Reply:
Sounds good!